tests/testthat/test-vec_fmt.R

```diff
@@ -6147,3 +6092,90 @@ test_that("The `vec_fmt_datetime()` function works", {
       )
     )
 })
+
+test_that("The `vec_fmt_duration()` function works", {
+  tm <- as.difftime(c("0:3:20"))
+  # tests are mostly duplicates of `fmt_duration()`
+  expect_equal(
+    vec_fmt_duration(tm),
+    "3m 20s"
+  )
+})
+
+test_that("vec_fmt_*() error when bad input are supplied.", {
```

Comment:
These tests are likely a bit redundant? but I guess it is better to keep them for safety?

Reply:
At this point, I'm generally okay with redundant tests.